lib/cli/options_t.ts

```diff
@@ -0,0 +1,7 @@
+export interface MochaArgs {
+  /** Path to `package.json` or `false` to skip */
+  config?: string | false;
+  package?: boolean;
+  /** Filepath to mocha.opts; defaults to whatever's in `mocharc.opts`, or `false` to skip */
+  opts?: string | false;
+}
```

Comment:
please do this with a `@typedef`; I really don't want to add any typescript sources.